.github/workflows/deploy-dev.yml

```
@@ -74,13 +74,15 @@ jobs:
           ECR_REGISTRY: ${{ secrets.ECR_REGISTRY }}
           ECR_REPOSITORY: nhsx-standards-directory
           IMAGE_TAG: ui-${{ github.sha }}
+          PAGES_CKAN_URL: 'https://manage.test.standards.nhs.uk/api/action'
         with:
           command: >-
             helm upgrade --install --wait -n dev ui ./charts/ui
             --set ui.config.ckanurl='${{ secrets.CKAN_URL }}'
             --set ui.config.tracking_id='${{ secrets.TRACKING_ID }}'
             --set ui.config.google_tag_id='${{ secrets.GOOGLE_TAG_ID }}'
             --set ui.container.image=$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+            --set ui.config.pagesckanurl='https://manage.test.standards.nhs.uk/api/action'
```

The env var is already set on line 77 so this can be:
```suggestion
            --set ui.config.pagesckanurl=$PAGES_CKAN_URL
```